bims/tasks/water_temperature.py

```diff
@@ -133,6 +130,12 @@ def process_water_temperature_data(
                     query = None
                     is_data_exists = False
 
+                if not water_temp_value:
+                    if is_data_exists:
+                        if query.value < 0:
```

Comment:
@Fanevanjanahary I think we don't need check if the value is less than 0, just remove the water temperature data if it exists.